### PR TITLE
Use core location if possible

### PR DESCRIPTION
**Pull request by ochronus**

### CODE REVIEW

**Comment by ochronus** [comment]
Ignore the first commit plz

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ googlemaps==2.4.3
 keyring==9.0
 requests==2.9.1
 wsgiref==0.1.2
+beautifulsoup4==4.4.1
```
Comment (akoshochrein) [inline requirements.txt]: What do you need soup4 for?

Comment (ochronus) [inline requirements.txt]: It remained there from the directions solution, removed for now

**Comment by akoshochrein** [comment]
👍

**Pull request merged**